toolbox/strings.py

```diff
@@ -102,6 +178,55 @@ def is_invite(string: str, *, fullmatch: bool = True) -> bool:
     return False
 
 
+def remove_markdown(content: str, formats: MarkdownFormat = MarkdownFormat.ALL) -> str:
+    """
+    Removes the markdown formatting from Discord messages.
+
+    Parameters
+    ----------
+    content : str
+        The `str` object, which needs their content cleaned from Discord's markdown formatting.
+    formats : MarkdownFormat
+        The `IntFlag` of the formatting that needs to be removed. Default is `MarkdownFormat.ALL`
+
+    Returns
+    -------
+    str
+        The cleaned string without markdown formatting.
+    """
+    for format, (regex, replace) in FORMAT_DICT.items():
+        if formats & format:
+            if format == MarkdownFormat.MULTI_QUOTE or format == MarkdownFormat.QUOTE:
+                content = remove_quote(content, format)
+            matches = re.findall(regex, content)
+            for match in matches:
+                content = re.sub(replace.format(match), match, content)
+    return content
+
+
+def remove_quote(content: str, formats: MarkdownFormat) -> str:
```

Comment:
I noticed you tried to typehint this differently earlier but had to change it; you use utilise the `Literal` type to specify individual formats that can be passed here.

Reply:
Ooooooh this is actually great to know. I had to yeet it because of mypy. I will keep that in mind!